packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -1043,20 +1043,6 @@ class WebpackCLI implements IWebpackCLI {
         description: "Stop webpack-cli process with non-zero exit code on warnings from webpack.",
         helpLevel: "minimum",
       },
-      // TODO remove this in the next major release, because not all webpack versions have this flag in CLI options
-      {
-        name: "extends",
```

Comment:
This will come from webpack schema since https://github.com/webpack/webpack/releases/tag/v5.82.0